### PR TITLE
Added V0 Pi 0 Mount

### PR DESCRIPTION
**Pull request by johnderuntz**
Created KenadyDwag44 Folder and added V0 Pi 0 mounting bracket.

Tested by myself and uses no new hardware.

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod.  Can you update `printer_mods/README.md` to add your mod to the table there in alphabetical order?  Also, if you have any pictures of the mod, feel free to include them.

Checked:
- mod in correct directory
- no spaces in directory or file names
- there's a readme for the mod
- STL is oriented correctly and doesn't require supports

**Comment by johnderuntz** [comment]
Updated the README to add my information!

**Review by KiloQubit — APPROVED** [review]
Looks good, thanks.

File: printer_mods/KenadyDwag44/sw_lightbar/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Voron V0 Raspberry Pi 0 Mount
```
Comment (KiloQubit) [inline printer_mods/KenadyDwag44/sw_lightbar/README.md:1]: This readme talks about a Pi Zero bracket, but the mod is for an LED holder, please update this README to describe the LED holder mod.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Please fix the orientation on the SW_lightbar* STLs, they're not in a printable orientation.

**Comment by johnderuntz** [comment]
Sorry, accidentally pushed the wrong file into the wrong pull request. Closing pull request because pull request #190 is the actual request

**Comment by johnderuntz** [comment]
Reopening

**Review by KiloQubit — APPROVED** [review]
Cleaned up PR looks good.

**Review by raymondh2 — APPROVED** [review]

**Review by FHeilmann — CHANGES_REQUESTED** [review]
This mod is kind of doing the same thing as [this mod](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Nomsplease/V0_Mods/pi_zero_camera_mount). Are there any significant advantages of this mod?

**Comment by johnderuntz** [comment]
> This mod is kind of doing the same thing as [this mod](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Nomsplease/V0_Mods/pi_zero_camera_mount). Are there any significant advantages of this mod?

Sorry, I didn't realize that mount existed before i submitted mine.

The main advantage of my mount compared to the other is that it is smaller and uses half the filament the other mod uses, but serves the purpose just as good. It was made smaller in order to fit a usb expansion board on the pi 0.

Thanks!

**Comment by KiloQubit** [comment]
Closing as a duplicate mod.

**Pull request closed without merging**